File: result/translate/transcoder/java2python/python/COUNT_POSSIBLE_PATHS_TOP_LEFT_BOTTOM_RIGHT_NXM_MATRIX_2.py
def f_gold ( m , n ) :
    dp = np.zeros ( ( n , m ) )
    dp [ 0 ] = 1
    for i in range ( m ) :
        for j in range ( 1 , n ) :
            dp [ j ] += dp [ j - 1 ]
    return dp [ n - 1 ]

File: result/repair/transcoder/python2java/java_repair/COUNT_POSSIBLE_PATHS_TOP_LEFT_BOTTOM_RIGHT_NXM_MATRIX_2.java
  int f_gold ( int p , int q ) {
  int [ ] dp = new int [ q ] ;
  dp [ 0 ] = 1 ;
for (int i=0;i<p;i++){
    for ( int j = 1 ;
    j < q ;
    j ++ ) {
      dp [ j ] += dp [ j - 1 ] ;
    }
  }
  return dp [ q - 1 ] ;
}
